### ISSUE TITLE
Lock change accent color - user need to watch a Reward ad

### ISSUE THREAD

**Issue by TNelen** [TNelen/BlackBox#252]

**Comment by Nuytemans-Dieter** [comment]
How would this work exactly?
- Unlock a new accent color (for them to keep) by watching an ad
- Watch an ad before you can change the accent color at all

**Comment by TNelen** [comment]
Yes first one! Give them option between two colors, but lock 2 or more option with ad.

But first we need to clarify our overall ad strategy.

**Comment by Nuytemans-Dieter** [comment]
That seems like a good idea.
